src/component/landing/index.jsx

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-unused-vars */ /* DELETE THIS ENTIRE LINE WHEN WE USE THE VARIABLE */
```

Comment:
Also, please don't delete this line unless we have used `uberPrices` and `lyftPrices` variables or else it will throw an error.